app/uk/gov/hmrc/ukimauthcheckerfrontend/config/AppConfig.scala

```diff
@@ -7,4 +23,5 @@ import play.api.Configuration
 class AppConfig @Inject()(config: Configuration) {
   val welshLanguageSupportEnabled: Boolean = config.getOptional[Boolean]("features.welsh-language-support").getOrElse(false)
 
+  val betaFeedbackUrl: String = "/"
```

Comment:
Can we add lemonlabs.url and return urls if we're going to have? (As much as this is just a root url)